docs/field-values-usage.asciidoc

```diff
@@ -0,0 +1,171 @@
+
+[float]
+==== Firewall blocking a network connection
+
+This event from a firewall describes a successfully blocked network connection:
+
+[source,json]
+----
+...
+  {
+    "source": {
+      "address": "10.42.42.42",
+      "ip": "10.42.42.42",
+      "port": 38842
+    },
+    "destination": {
+      "address": "10.42.42.1",
+      "ip": "10.42.42.1",
+      "port": 443
+    },
+    "rule": {
+      "name": "wan-lan",
+      "id": "default"
+    },
+    ...
+    "event": {
+      "kind": "event", <1>
+      "category": [ <2>
+        "network"
+      ],
+      "type": [ <3>
+        "connection",
+        "denied"
+      ],
+      "outcome": "success", <4>
+      "action": "dropped" <5>
+    }
+  }
+...
+----
+
+<1> Classifying as an `event`.
+<2> `event.category` categorizes this event as `network` activity.
+<3> The event was both an attempted network `connection` which was `denied`.
+<4> The blocking of this connection is expected. The outcome is a `success` from the perspective of the firewall emitting the event.
+<5> The firewall classifies this denied connection as `dropped`, and this value is captured in `event.action`.
+
+A "denied" network connection could fall under different action values: "blocked", "dropped", "quarantined", etc. The `event.action` field captures the action taken as described by the source, and populating `event.type:denied` provides an independent, normalized value.
+
+A single query will return all denied network connections which have been normalized with the same categorization values:
+
+[source,sh]
+----
+event.category:network AND event.type:denied
+----
+
+[float]
+==== Failed attempt to create a user account
+
+User `alice` attempts to add a user account, `bob`, into a directory service, but the action fails:
+
+[source,json]
+----
+{
+  "user": {
+    "name": "alice",
+    "target": {
+      "name": "bob"
+    }
+  },
+  "event": {
+    "kind": "event", <1>
+    "category": [ <2>
+      "iam"
+    ],
+    "type": [ <3>
+      "user",
+      "creation"
+    ],
+    "outcome": "failure" <4>
+  }
+}
+----
+
+<1> Again classifying as an `event`.
+<2> Categorized using `iam` for an event user account activity.
+<3> Both `user` and `creation`
+<4> The creation of a user account was attempted, but it was not successful.
+
+[float]
+==== Informational listing of a file
+
+A utility, such as a file integrity monitoring (FIM) application, takes inventory of a file but does not access or modify the file:
+
+[source,json]
+----
+{
+  "file": {
+    "name": "example.png",
+    "owner": "alice",
+    "path": "/home/alice/example.png",
+    "type": "file"
+  },
+  "event": {
+    "kind": "event",
+    "category": [ <1>
+      "file"
+    ],
+    "type": [ <2>
+      "info"
+    ]
+  }
+}
+----
+
+<1> The event is reporting on a `file`.
+<2> The `info` type categorizes purely informational events. The target file here was not accessed or modified.
+
```

Comment:
might want to follow same format as others where

<1> is event.kind
<2> is event.category
<3> is event.type
<4> is event.outcome

and for this is example it might be good to say why event.outcome wasn't populated. (no data I assume)